Tasks/AzureAppConfigurationPushV10/task.json

```diff
@@ -0,0 +1,305 @@
+{
+    "id": "04fc3ed1-4513-4f38-b0fe-5a15864bfa9c",
+    "name": "AzureAppConfigurationPush",
+    "friendlyName": "Azure App Configuration Push",
+    "description": "Push key-values to an Azure App Configuration instance",
+    "helpUrl": "https://learn.microsoft.com/azure/azure-app-configuration/push-kv-devops-pipeline",
+    "helpMarkDown": "Email AzureAppConfig@microsoft.com for questions.",
+    "category": "Utility",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 10,
+        "Minor": 244,
```

Comment:
This should be [245](https://whatsprintis.it/) now

Reply:
Thanks this has been updated to 245